core/domain/translatable_object_registry_test.py

```diff
@@ -24,44 +24,46 @@
 class TranslatableObjectRegistryUnitTests(test_utils.GenericTestBase):
     """Test the Registry class in translatable_object_registry."""
 
-    def test_get_object_class_method(self):
+    def test_get_object_class_method(self) -> None:
         """Tests the normal behavior of get_object_class()."""
         retrieved_class = (
             translatable_object_registry.Registry.get_object_class(
                 'TranslatableHtml'))
         self.assertEqual(retrieved_class.__name__, 'TranslatableHtml')
 
-    def test_nontranslatable_class_is_not_gettable(self):
+    def test_nontranslatable_class_is_not_gettable(self) -> None:
         """Tests that trying to retrieve a non-translatable class raises an
         error.
         """
-        with self.assertRaisesRegex(
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
             TypeError, 'not a valid translatable object class'):
             translatable_object_registry.Registry.get_object_class(
-                'Int')
+                    'Int') # type: ignore[arg-type]
```

Comment:
```suggestion
                'Int') # type: ignore[arg-type]
```

Reply:
Here i used the ignore particularly because it is a test case for catching wrong inputs . Thus we are explicitly passing wrong data to it . So will there be need for TODO ?

Reply:
Yes, we should get rid of these checks once everything is typed. There is already an issue for that.